sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcIO.java

```diff
@@ -799,6 +810,11 @@ public ReadRows withOutputParallelization(boolean outputParallelization) {
       return toBuilder().setOutputParallelization(outputParallelization).build();
     }
 
+    /** Whether to disable auto commit on read. Defaults to true if not provided. */
```

Comment:
Could you please elaborate javadoc. For example, noting that some Jdbc provider does not support setAutoCommit, which is the circumstance one needs to set this to false

Reply:
done